src/docs/cursor-log.dox

```diff
@@ -25,11 +25,7 @@ To open a log cursor on the database:
 @snippet ex_log.c log cursor open
 
 A log cursor's key is a unique log record identifier, plus a
-@m_if{c}
 uint32_t
```

Comment:
While you're in here, please change this to `\c uint32_t`

Reply:
Fixed.